src/AuthContext.tsx

```diff
@@ -251,7 +257,9 @@ export const AuthProvider = ({ authConfig, children }: IAuthProvider) => {
   }, []) // eslint-disable-line
 
   return (
-    <AuthContext.Provider value={{ token, tokenData, idToken, idTokenData, login, logOut, error, loginInProgress }}>
+    <AuthContext.Provider
+      value={{ token, tokenData, idToken, idTokenData, login, logOut, error, loginInProgress }}
+    >
```

Comment:
Prettier formatting:
```suggestion
  return (
    <AuthContext.Provider value={{ token, tokenData, idToken, idTokenData, login, logOut, error, loginInProgress }}>
      {children}
```